### DISCUSSION TITLE
Backup scheme in case of EL18 touch screen failure

### DISCUSSION THREAD

**Discussion by AtherisFPV** [EdgeTX/edgetx#2952] [Ideas]
The use of EL18 mainly depends on the touch screen, but at present, the touch screen function is not easy to use, and there are problems such as jamming and failure. 
Since EL18 has two five dimensional buttons, I would like to propose an alternative operation scheme: set an external switch. 
When the switch is not triggered, the two five dimensional buttons are still the current functions, fine tune the channel, confirm and return.
However, when the switch is triggered, the two five dimensional buttons will realize the function of global menu buttons, just like the operation in the LUA script, but now it will be magnified to the global. Operational logic available for the entire system.

I wonder if this scheme is feasible?
I hope to provide you with an idea, thank you!

**Comment by pfeerick** [comment]
This is something that had been recently discussed again on the project Discord. Originally I had intended to make it so if you pressed both trim hats down, they would switch between trim and navigation modes. However, that was deemed problematic, so my next train of thought was an option of "trims/navigation/switchable", where when you select the last option you can assign it to a switch. So we're clearly on the same track there! :)

**Reply by AtherisFPV** [reply]
Wow, that's great. I really hope you can do it! Thank you very much, bro！When there is a problem with the touch screen, we can still use the EdgeTX system normally！😁👍
